includes/class-wc-pakettikauppa-shipping-method.php

```diff
@@ -76,14 +76,11 @@ public function __construct( $instance_id = 0 ) {
        */
       public function init() {
         $settings = $this->wc_pakettikauppa_shipment->get_settings();
-        if ( ! isset($settings['show_pakettikauppa_shipping_method']) ) {
-          $show_pakettikauppa_shipping_method = 'yes';
-        } else {
-          $show_pakettikauppa_shipping_method = $settings['show_pakettikauppa_shipping_method'];
-        }
+        $show_method_set = isset($settings['show_pakettikauppa_shipping_method']);
+        $show_pakettikauppa_shipping_method = ! $show_method_set ? 'yes' : $settings['show_pakettikauppa_shipping_method'];
 
```

Comment:
I think this might have a if (empty($settings)) to set that to 'no'.

Reply:
That's would just bloat the code, it doesn't add any value. It's the same code from earlier with less if-else. 

It could be `no` by default, but it doesn't matter. It's going to get refactored anyway.

Reply:
That new code afaik works as following:
is show_pakettikauppa_shipping_method is not set, $show_method_set == false

next one says:
if $show_method_set == false, then set $show_pakettikauppa_shipping_method to 'yes' otherwise use the value in the store.

But if show_pakettikauppa_shipping_method is not set, we have to see if there is any Pakettikauppa shipping methods in use. And if there is, then set it to yes -> otherwise set it to no.

But we can add a check before that to see if $settings is empty, then we can set it as 'no' without checking the shipping methods in the zones.

Reply:
If the settings is empty, then we don't need to save the settings and we can show the wizard.

Reply:
Yeah. I'll address that in the refactor.